Docs/AcpiSamples/SSDT-EC-USBX.dsl

```diff
@@ -53,32 +53,52 @@ DefinitionBlock ("", "SSDT", 2, "ACDT", "SsdtEC", 0x00001000)
     }
     **/
 
+    /*
+     * Uncomment the following to disable EC if the STA method exists.
+     */
+    
+    /**
+    External (_SB_.PCI0.LPCB.EC0_._STA, UnknownObj)
+    
+    Scope (\)
+    {
+        If (_OSI ("Darwin"))
+        {
+            \_SB.PCI0.LPCB.EC0._STA = Zero
+        }
+        
+    }    
+    **/
+
     Scope (\_SB)
     {
-        Device (USBX)
+        If (_OSI ("Darwin"))
```

Comment:
All good, but this check should be in _STA.

Reply:
ok